libraries/botbuilder-core/src/activityHandler.ts

```diff
@@ -3,17 +3,11 @@
  * Licensed under the MIT License.
  */
 
-import {
-    Activity,
-    AdaptiveCardInvokeResponse,
-    AdaptiveCardInvokeValue,
-    MessageReaction,
-} from 'botframework-schema';
-
+import { Activity, AdaptiveCardInvokeResponse, AdaptiveCardInvokeValue, MessageReaction } from 'botframework-schema';
 import { ActivityHandlerBase } from './activityHandlerBase';
 import { InvokeException } from './invokeException';
 import { InvokeResponse } from './invokeResponse';
-import { StatusCodes } from './statusCodes';
+import { StatusCodes } from 'botframework-schema';
```

Comment:
This looks like an unintended side effect from the rebase, can you move this into the import on [Line 6](https://github.com/microsoft/botbuilder-js/pull/3728/files#diff-73672579d87cdfbc0d4fe836801679f0e66c122783b3b28781a95ded45580882R6)?